hydroflow/src/util/mod.rs

```diff
@@ -1,8 +1,11 @@
 //! Helper utilities for the Hydroflow surface syntax.
 
 mod udp;
+#[cfg(not(target_arch = "wasm32"))]
 pub use udp::*;
+
 mod tcp;
+#[cfg(not(target_arch = "wasm32"))]
 pub use tcp::*;
```

Comment:
Probably we ought to be a bit more principled with organization for these things that need to be turned on/off